frontend/packages/dev-console/src/components/edit-application/EditApplication.tsx

```diff
@@ -137,15 +137,15 @@ const EditApplication: React.FC<EditApplicationProps> = ({
     appResources.buildConfig.data,
     appResources.buildConfig.loaded,
     imageStreamsData,
-    pageHeading,
+    editFlowType,
   ]);
 
   const renderForm = (formikProps: FormikProps<any>) => (
     <EditApplicationForm
       {...formikProps}
       appResources={appResources}
       enableReinitialize
-      createFlowType={pageHeading}
+      createFlowType={editFlowType}
```

Comment:
Maybe we should remove the create and edit prefix? createFlowType=editFlowType looks strange or? I don't know how big the change is. The actual version is also fine for me.